src/stories/components/events/NewEvent.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import dayjs from "dayjs";
+import Typography from "@mui/material/Typography";
+import Stack from "@mui/material/Stack";
+import Image from "next/image";
+import CalendarIcon from "@/../public/images/Calendar.svg";
+import PlaceMarkIcon from "@/../public/images/PlaceMark.svg";
+
+interface NewEventProps {
```

Comment:
Add documentation Check  [ButtonProps interface](https://github.com/The-Product-Lounge/my-seat/blob/02e76331f920b84533f62b2b1481ae52b69510e8/src/lib/components/inputs/Button.component.tsx#L16C1-L16C1) for an example